### PR TITLE
Update directory names for global workflow

### LINKED ISSUE #897: Update directory variables for global workflow
Replace the use of `FIX*` directory variables with the full path and `FIXgfs`. Example: `FIXam` -> `FIXgfs/am`

Remove any obsolete variables, such as `OROFIX` and `FIX_SFC`.

Supports the global workflow: https://github.com/NOAA-EMC/global-workflow/issues/2184

### PR DESCRIPTION
**Pull request by GeorgeGayno-NOAA**
## DESCRIPTION OF CHANGES: 
Update directory names in the ice blend, snow and global_cycle scripts to be consistent with the latest global_workflow convention.

## TESTS CONDUCTED: 

- [x] Run relevant consistency tests locally on all Tier 1 machines. For this issue, these include the `global_cycle`, `ice_blend` and `snow2mdl` tests. See below for results.

## DEPENDENCIES:
https://github.com/NOAA-EMC/global-workflow/issues/2184

## DOCUMENTATION:
N/A

## ISSUE: 
Fixes #897.

## CONTRIBUTORS: 
@KateFriedman-NOAA

### CODE REVIEW

File: ush/global_cycle_driver.sh [hunk 1]
```diff
@@ -18,9 +18,8 @@ pwd=$(pwd)
 export NWPROD=${NWPROD:-$pwd}
```
Comment (GeorgeGayno-NOAA) [inline ush/global_cycle_driver.sh]: @KateFriedman-NOAA Are NWPROD and DMPDIR recognized by the workflow?

Comment (KateFriedman-NOAA) [reply ush/global_cycle_driver.sh]: `DMPDIR` is, but only for dev runs to provide the location of the global dump archive (GDA). `NWPROD` has been removed from the workflow. I still see `NWPROD` or similar in a few components, which should likely also be cleaned out.

Comment (GeorgeGayno-NOAA) [reply ush/global_cycle_driver.sh]: Should I remove `NWPROD` under this issue or not bother?

Comment (KateFriedman-NOAA) [reply ush/global_cycle_driver.sh]: I'd remove it. I'd be ok with it going in this issue.

Comment (GeorgeGayno-NOAA) [reply ush/global_cycle_driver.sh]: Kate, I removed other unused variables. See 0a5e76a.

File: ush/emcsfc_ice_blend.sh [hunk 2]
```diff
@@ -79,19 +79,18 @@ cd $DATA
 SENDCOM=${SENDCOM:-"NO"}
 
 envir=${envir:-"prod"}
-NWROOT=${NWROOT:-"/nw${envir}"}
 
-HOMEgfs=${HOMEgfs:-${NWPROD:-$NWROOT/gfs.${gfs_ver:?}}}
-FIXam=${FIXam:-$HOMEgfs/fix/am}
+HOMEgfs=${HOMEgfs:-/nw${envir}/gfs.${gfs_ver:?}}
```
Comment (KateFriedman-NOAA) [inline ush/emcsfc_ice_blend.sh]: The default for this should probably use the new `PACKAGEROOT` variable introduced in ops with the WCOSS2 port and which is set via the `prod_envir` module:
```
HOMEgfs=${HOMEgfs:-${PACKAGEROOT}/gfs.${gfs_ver:?}}
```

File: ush/emcsfc_snow.sh [hunk 3]
```diff
@@ -74,10 +74,9 @@ fi
 #------------------------------------------------------------------------
 
 envir=${envir:-"prod"}
-NWROOT=${NWROOT:-"/nw${envir}"}
-HOMEgfs=${HOMEgfs:-$NWROOT/gfs.${gfs_ver:?}}
+HOMEgfs=${HOMEgfs:-/nw${envir}/gfs.${gfs_ver:?}}
```
Comment (KateFriedman-NOAA) [inline ush/emcsfc_snow.sh]: Same comment regarding suggested use of `${PACKAGEROOT}` here.

File: ush/global_cycle.sh [hunk 4]
```diff
@@ -55,46 +53,46 @@
...
 #     FNZORC        Input roughness climatology.
 #                   Defaults to igbp vegetation type-based lookup table
 #                   FNVETC must be set to igbp file:
 #                   ${FIXam}/global_vegtype.igbp.t$JCAP_CASE.$LONB_CASE.$LATB_CASE.rg.grb
 #     FNALBC        Input 4-component albedo climatology GRIB file.
-#                   defaults to ${FIXam}/global_snowfree_albedo.bosu.t$JCAP_CASE.$LONB_CASE.$LATB_CASE.rg.grb
+#                   defaults to ${FIX_SFC}/${CASE}.mx${OCNRES}.snowfree_albedo.tileX.nc
 #     FNALBC2       Input 'facsf' and 'facwf' albedo climatology GRIB file.
-#                   Defaults to ${FIXam}/global_albedo4.1x1.grb
+#                   Defaults to ${FIX_SFC}/${CASE}.mx${OCNRES}.facsf.tileX.nc
```
Comment (KateFriedman-NOAA) [inline ush/global_cycle.sh]: I've also done away with `FIX_SFC` and `FIXsfc` on the global workflow side. I replaced it with `${FIXgfs}/orog/${CASE}/sfc`. So line 72 would be:
```
${FIXgfs}/orog/${CASE}/sfc/${CASE}.mx${OCNRES}.facsf.tileX.nc
```

File: ush/global_cycle.sh [hunk 5]
```diff
@@ -233,12 +231,11 @@ OCNRES=${OCNRES:-100}
...
-HOMEgfs=${HOMEgfs:-$BASEDIR/gfs_ver.${gfs_ver}}
+envir=${envir:-prod}
+HOMEgfs=${HOMEgfs:-/nw${envir}/gfs_ver.${gfs_ver}}
 EXECgfs=${EXECgfs:-$HOMEgfs/exec}
-FIX_DIR=${FIX_DIR:-$HOMEgfs/fix}
-FIXam=${FIXam:-$FIX_DIR/am}
-OROFIX=${OROFIX:-$FIX_DIR/orog/${CASE}}
+FIXgfs=${FIXgfs:-$HOMEgfs/fix}
+OROFIX=${OROFIX:-$FIXgfs/orog/${CASE}}
 FIX_SFC=${FIX_SFC:-$OROFIX/sfc}
```
Comment (KateFriedman-NOAA) [inline ush/global_cycle.sh]: I removed both `OROFIX` and `FIX_SFC` on the global-workflow side. See comment about `FIX_SFC` replacement above.

File: ush/global_cycle_driver.sh [hunk 6]
```diff
@@ -15,12 +15,12 @@ export CDUMP=${CDUMP:-gfs}                   # gfs or gdas
...
-export NWPROD=${NWPROD:-$pwd}
-export DMPDIR=${DMPDIR:-$NWPROD}
-export HOMEgfs=${HOMEgfs:-$NWPROD/gfs.v15.0.0}
-export FIX_DIR=${FIX_DIR:-$HOMEgfs/fix}   
-export FIXam=${FIXam:-$FIX_DIR/am}   
-export OROFIX=${OROFIX:-$FIX_DIR/orog/${CASE}}
+export DMPDIR=${DMPDIR:-$pwd}
+export envir=${envir:-prod}
+export gfs_ver=${gfs_ver:-v15.0.0}
+export HOMEgfs=${HOMEgfs:-/nw${envir}/gfs.${gfs_ver}}
```
Comment (KateFriedman-NOAA) [inline ush/global_cycle_driver.sh]: Similar comment about suggestion to introduce `${PACKAGEOOT}` here.

File: ush/global_cycle_driver.sh [hunk 7]
```diff
@@ -15,12 +15,12 @@ export CDUMP=${CDUMP:-gfs}                   # gfs or gdas
...
+export FIXgfs=${FIXgfs:-$HOMEgfs/fix}   
+export OROFIX=${OROFIX:-$FIXgfs/orog/${CASE}}
 export FIX_SFC=${FIX_SFC:-$OROFIX/sfc}
```
Comment (KateFriedman-NOAA) [inline ush/global_cycle_driver.sh]: Similar comment as above.

**Review by KateFriedman-NOAA — COMMENTED** [review]
Looks good, lots of `FIX*` variables cleaned out and replaced with fuller paths. Left a few comments with suggestions to consider before I approve.

File: ush/emcsfc_ice_blend.sh (hunk 2 above)
Comment (GeorgeGayno-NOAA) [reply ush/emcsfc_ice_blend.sh]: What is the default value for PACKAGEROOT?

Comment (KateFriedman-NOAA) [reply ush/emcsfc_ice_blend.sh]: It's:

```
kate.friedman@dlogin02:~> module load prod_envir
kate.friedman@dlogin02:~> echo $PACKAGEROOT
/lfs/h1/ops/prod/packages
``

Comment (GeorgeGayno-NOAA) [reply ush/emcsfc_ice_blend.sh]: Done at 733996b.

File: ush/emcsfc_snow.sh (hunk 3 above)
Comment (GeorgeGayno-NOAA) [reply ush/emcsfc_snow.sh]: See 733996b.

File: ush/global_cycle.sh (hunk 4 above)
Comment (GeorgeGayno-NOAA) [reply ush/global_cycle.sh]: Done at 97b765e.

File: ush/global_cycle_driver.sh (hunk 6 above)
Comment (GeorgeGayno-NOAA) [reply ush/global_cycle_driver.sh]: Done.

File: ush/global_cycle_driver.sh (hunk 7 above)
Comment (GeorgeGayno-NOAA) [reply ush/global_cycle_driver.sh]: Done.

File: ush/global_cycle.sh (hunk 5 above)
Comment (GeorgeGayno-NOAA) [reply ush/global_cycle.sh]: Done at 97b765e.

File: ush/emcsfc_ice_blend.sh (hunk 2 above)
Comment (GeorgeGayno-NOAA) [reply ush/emcsfc_ice_blend.sh]: Forgot to add PACKAGEROOT to ./scripts/exemcsfc_global_sfc_prep.sh

**Review by KateFriedman-NOAA — APPROVED** [review]
Changes look good, thanks @GeorgeGayno-NOAA !

File: ush/emcsfc_ice_blend.sh (hunk 2 above)
Comment (GeorgeGayno-NOAA) [reply ush/emcsfc_ice_blend.sh]: Done at 01f43b6.

Comment (KateFriedman-NOAA) [reply ush/emcsfc_ice_blend.sh]: Thanks!

**Comment by GeorgeGayno-NOAA** [comment]
Using 7c8cb56, the `global_cycle`, `ice_blend` and `snow2mdl` consistency tests were run on Hera, Jet, Orion, Hercules and Dogwood. All tests passed as expected. 

Because of the new 'fixed' file variables, the `global_cycle` input fixed directory had to be rearranged a bit. The 'case' directories were moved to a new `orog` sub-directory. And so previous snapshots of 'develop' would still work, links were made to point at the old location.

```
dlogin02@ /lfs/h2/emc/nems/noscrub/emc.nems/UFS_UTILS/reg_tests/global_cycle/fix\ $ ls -lr
total 4
dr-xr-sr-x 5 emc.nems nems 4096 Feb 21 20:04 orog
lrwxrwxrwx 1 emc.nems nems   11 Feb 21 20:04 C768 -> ./orog/C768
lrwxrwxrwx 1 emc.nems nems   10 Feb 21 20:04 C48 -> ./orog/C48
lrwxrwxrwx 1 emc.nems nems   11 Feb 21 20:04 C192 -> ./orog/C192
```

**Pull request merged**